### DISCUSSION TITLE
How to Apply `group-hover` Styles from Parent to Children?

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#14330] [Help] [answered]
Is there a way to use Tailwind CSS's group-hover to apply styles from parent element to children?
```
 <div class="relative w-fit group group-hover:[&_.content]:visible">
      <div class="content"></div>
</div>
```
If not, what are the best workarounds to achieve this effect in Tailwind

**Comment by wongjn** [comment, marked as answer]
Could elaborate more on your question please? I don't think I quite understand the need to use `group-hover` in this situation.

If you want the top-level `<div>` to have `visibility: visible` when it is hovered, you'd do:

```html
<div class="relative w-fit hover:visible">
```

If you want `.content` to have `visibility: visible` when to top-level `<div>` is hovered, you'd do:

```html
<div class="relative w-fit group">
  <div class="content group-hover:visible">
```

Or if you can't apply classes to the `content` element, you could do:

```html
<div class="relative w-fit [&:hover_.content]:visible">
  <div class="content">
```
Or

```html
<div class="relative w-fit [&_.content]:hover:visible">
  <div class="content">
```

**Reply by MonaAghili** [reply]
Thanks you.
